README.md

```diff
@@ -1 +1,132 @@
-# Book-Store
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <h3><b>Book Store</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 math-magician <a name="about-project"></a>
+
+> This is the Book-Store project for react JS.
+
+**math-magician** is a...
```

Comment:
- [ ] [OPTIONAL] Well done for using the latest README template 👏 Please ensure that the title of your project is correct. It is currently called "math-magician" instead of "bookstore" 👍 
![image](https://user-images.githubusercontent.com/46839548/213849046-0b65781c-55d2-47e6-a3b5-dac8ec0c4899.png)